src/qt_frontend/n64_emulator_thread.cpp

```diff
@@ -32,21 +35,18 @@ N64EmulatorThread::N64EmulatorThread(QtWSIPlatform* wsiPlatform) {
     }
 }
 
-void N64EmulatorThread::start() {
+void N64EmulatorThread::run() noexcept {
     if (n64_should_quit() || running) {
         logfatal("Tried to start emulator thread, but it was already running!");
     }
 
-    QtWSIPlatform* _wsiPlatform = this->wsiPlatform;
-    QWindow* pane = wsiPlatform->getPane();
     running = true;
-    bool* _game_loaded = &game_loaded;
-    emuThread = std::thread([_wsiPlatform, pane, _game_loaded]() {
-        init_vulkan_wsi(_wsiPlatform, std::make_unique<QtParallelRdpWindowInfo>(pane));
+    emuThread = std::thread([&] {
```

Comment:
This doesn't need to run in yet another separate thread. N64EmulatorThread has become the thread itself and its run function is part of the abstract QThread class. It will already be run in another thread.

Reply:
Fixed. Works as expected